modules/fedimint-mint-client/src/lib.rs

```diff
@@ -377,6 +382,60 @@ impl OOBNotes {
     }
 }
 
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Serialize, Deserialize, Encodable, Decodable)]
+pub struct OOBNoteV2 {
+    pub amount: Amount,
+    pub sig: Signature,
+    pub key: KeyPair,
+}
+
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize, Encodable, Decodable)]
+pub struct OOBNotesV2 {
+    pub mint: InviteCodeV2,
```

Comment:
The `mint` field will be encoded as JSON and not as a base64 string, right?

Reply:
Yes.